src/main/java/com/faforever/client/config/BaseConfig.java

```diff
@@ -49,10 +51,19 @@ Locale locale() {
   MessageSource messageSource() {
     ResourceBundleMessageSource messageSource = new ResourceBundleMessageSource();
     messageSource.setBasename("i18n.messages");
+    messageSource.setFallbackToSystemLocale(false);
```

Comment:
Hmm that should not happen anyway, but if u set it true it just ignores everything else that you tell it.